build.gradle

```diff
@@ -94,11 +96,11 @@ defaults {
         [id: 'jmoore', name: 'Jim Moore', email: 'moore.jim@gmail.com'],
     ]
 
-    copyrightYears = '2014-2015'
+    copyrightYears = '2014-2017'
 }
 
 wrapper {
-    gradleVersion = 2.9
+    gradleVersion = "3.5"
```

Comment:
No, the "raw" `2.9` or `3.5` are treated by the Groovy parser as floats and then cast into Strings. Where it runs into trouble is when you try to do a version like `2.4.1` or `3.0-RC1`, which do not parse except as Strings. So I just "preemptively" declared it as a String.